staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -2343,26 +2342,16 @@ func TestValidateCustomResourceDefinition(t *testing.T) {
 													Properties: map[string]apiextensions.JSONSchemaProps{
 														"name": {
 															Type: "string",
-															// TODO: forbid wrongly typed nested fields in metadata
-															Default: jsonPtr("%"),
```

Comment:
do we also have coverage of a correctly-typed non-validating default like `%` for metadata.name?

Reply:
yes, should be below, in multiple variations.

Reply:
search for "invalid-in-object-defaults"